src/promptflow-devkit/promptflow/_sdk/_orchestrator/test_submitter.py

```diff
@@ -111,6 +112,8 @@ def flow(self):
     @property
     def dataplane_flow(self):
         # TODO: test submitter shouldn't interact with dataplane flow directly
```

Comment:
looks like we will always do a deserialize? check not _data_plane_flow here also

Reply:
fix it